rfcs/0146-meta-categories.md

```diff
@@ -144,6 +147,32 @@ A typical snippet of `lib.categories` will be similar to:
 
 ```
 
+### Hybrid approach
+[hybrid-approach]: #hybrid-approach
+
+A hybrid approach for code implementation would be implement two meta
+attributes, namely
+
+- `meta.categories` for Appstream-based categories
+  -  the corresponding `lib.categories` should follow Appstream closely, with
+     few room to custom/extra categories
+- `meta.tags` for Debtags-like tags
```

Comment:
I think we have discussed that as categories are sometimes a list anyway, «tags» can just got into the end of the same list?

Reply:
I will move this to alternatives.